### PR TITLE
Resolve logout issue

### PR DESCRIPTION
**Pull request by PichapopRo**
## What I have done
I have discovered that Logout is completely broken so I fixed it.

### CODE REVIEW

**Comment by Pat-7-626** [comment]
Login does not work after clicking logout and logged out then try to login again. (The button is still 'Login' as if I didn't log in after login and the URL is 'http://127.0.0.1:8000/signout/', has to click Home to go to http://127.0.0.1:8000/recipes/ and click 'Login' then login to make the button become 'Logout' meaning I logged in.)

**Comment by PichapopRo** [comment]
I fixed the URL problem now can you tell me more about "Login does not work after clicking logout and logged out then try to login again. (The button is still 'Login' as if I didn't log in after login"? I'm kind of confused because when I test it the button is logout when user logged in and when user logged out the button changes to Login.

**Comment by Pat-7-626** [comment]
> I fixed the URL problem now can you tell me more about "Login does not work after clicking logout and logged out then try to login again. (The button is still 'Login' as if I didn't log in after login"? I'm kind of confused because when I test it the button is logout when user logged in and when user logged out the button changes to Login.

Try login, then logout, then login again and look at the button's name, it's gonna still be 'Login' even tho you just logged in.

**Comment by SariyaP** [comment]
It works fine for me so I don't really see the issues here, can you elaborate or try again? Since the redirect was already fixed.

File: webpage/views.py
Comment (knilios) [inline webpage/views.py:1]: image

What happened here?

**Review by knilios — COMMENTED** [review]
Everything looks fine except for the weird spacing I've commented on.

**Comment by PichapopRo** [comment]
I fixed the weird spacing now. I think it's because I clicked Reformat in pycharm so it adjust the code weirdly.

**Review by knilios — APPROVED** [review]
Looks alright to me.

**Pull request merged**